wai-handler-hal.cabal

```
@@ -77,6 +77,7 @@ test-suite wai-handler-hal-tests
     , pretty-simple    ^>=4.1.0.0
     , tasty            >=1.3      && <1.6
     , tasty-golden     ^>=2.3
+    , tasty-hunit      >=0.9
```

```suggestion
    , tasty-hunit      >=0.9 && <0.11
```
(and then `cabal-fmt`)